marquez_client/http_backend.py

```diff
@@ -0,0 +1,62 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import requests
+import os
+
+from marquez_client.constants import (DEFAULT_TIMEOUT_MS)
+from marquez_client.backend import Backend
+from marquez_client import errors
+
+log = logging.getLogger(__name__)
+
+
+class HttpBackend(Backend):
+    def __init__(self, url):
+        self._timeout = self._to_seconds(
+            os.environ.get('MARQUEZ_TIMEOUT_MS', DEFAULT_TIMEOUT_MS))
```

Comment:
I would add a timeout parameter to init and move this the the factory.

Reply:
done.